### ISSUE TITLE
Shows discipline in upholding the team's engineering process

### ISSUE THREAD

**Issue by lisamburns** [pivotal-cf/areas-of-contribution#60]
I find this question somewhat ambiguous.
If it is "Is disciplined in personally following the team's engineering process", which I view as a P1 skill.
Or if it is meant to imply
"Notices when there is a deviation from the team's engineering process and holds the team accountable" which would be P2 or P3.

**Comment by rainmaker** [comment]
Good point @lisamcho , we can improve the wording here in the upcoming release. "Notices when there is a deviation from the team's engineering process and holds the team accountable" sounds good to me, or maybe "Holds the team accountable for following a healthy and agreed-upon engineering process"
